Apps/WebClient/src/Views/Shared/_Layout.cshtml

```diff
@@ -10,6 +10,29 @@
     <environment exclude="Development">
         <link rel="stylesheet" href="~/dist/site.css" asp-append-version="true" />
     </environment>
+
+    <!-- Snowplow starts plowing - Standalone vA.2.10.2 -->
+    <script type="text/javascript">
+        ;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];
+        p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)
+        };p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
+        n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,"script","https://sp-js.apps.gov.bc.ca/MDWay3UqFnIiGVLIo7aoMi4xMC4y.js","snowplow"));
+        var collector = 'spm.gov.bc.ca';

```

Comment:
I think I remember something about making this configurable, did we decide to keep it static?

Reply:
for now. We can instroduce a razor variable once we get there. But until we can check if we are actually hitting anything I dont think its necessary